packages/builder/src/pages/builder/portal/settings/ai/AIConfigTile.svelte

```diff
@@ -0,0 +1,130 @@
+<script>
+  import { Body, Label, Icon } from "@budibase/bbui"
+  import BudibaseLogo from "./logos/Budibase.svelte"
+  import OpenAILogo from "./logos/OpenAI.svelte"
+  import AnthropicLogo from "./logos/Anthropic.svelte"
+  import TogetherAILogo from "./logos/TogetherAI.svelte"
+  import { Providers } from "./constants"
+
+  export let config
+  export let disabled
+
+  export let editHandler
+  export let deleteHandler
+</script>
+
+<!-- svelte-ignore a11y-no-static-element-interactions -->
+<!-- svelte-ignore a11y-click-events-have-key-events -->
+<div on:click class:disabled class="option">
+  <div class="icon">
+    {#if config.name === "Budibase AI"}
+      <BudibaseLogo height="30" width="30" />
+    {:else if config.provider === Providers.OpenAI.name}
+      <OpenAILogo height="30" width="30" />
+    {:else if config.provider === Providers.Anthropic.name}
+      <AnthropicLogo height="30" width="30" />
+    {:else if config.provider === Providers.TogetherAI.name}
+      <TogetherAILogo height="30" width="30" />
+    {/if}
```

Comment:
It feels like there ought to be a structure under `config.provider` that includes a `config.provider.logo` to make this a bit more robust to change.

Reply:
Updated to use `svelte:component` - unfortunately you can't just pull in through a string, so just defined the logos above as constants and at least we don't have to keep extending this `if` statement